odoo/tests/common.py

```diff
@@ -655,11 +655,16 @@ def setUp(self):
         # restore environments after the test to avoid invoking flush() with an
         # invalid environment (inexistent user id) from another test
         envs = self.env.all.envs
+        for env in envs:
+            self.addCleanup(env.clear)
+        # `envs` is a `WeakSet` and we add the functions in a stack. So thanks
+        # to the two following lines, we will actually:
+        # 1. clear the set
+        # 2. update it with the existing env of `envs`
```

Comment:
The comment is useless IMHO, as it only describes what the code does.
```suggestion
        # restore the set of known environments as it was at setUp
```